src/components/character-count/character-count.njk

```diff
@@ -0,0 +1,10 @@
+{% from "character-count/macro.njk" import govukCharacterCount %}
+
+{{ govukCharacterCount({
+  id: "more-detail",
+  name: "more-detail",
+  label: {
+    text: "Can you provide more detail?",
+    hintText: "Don't include personal or financial information, eg your National Insurance number or credit card details."
+  }
+}) }}
```

Comment:
I don't think we have this file for other components. Does it help to demonstrate how you would use the component?

Reply:
Example files were removed in #890 – I imagine the work on this started before that change was made.

Reply:
Wasn't aware. Will check and update.